addons/project/controllers/portal.py

```diff
@@ -94,40 +93,102 @@ def portal_my_project(self, project_id=None, access_token=None, **kw):
     # My Task
     # ------------------------------------------------------------
     def _task_get_page_view_values(self, task, access_token, **kwargs):
+        try:
+            project_accessible = task.project_id.id and self._document_check_access('project.project', task.project_id.id)
```

Comment:
Oups, my bad, since the _document_check_access can return a recordset then the project_accessible will not be boolean value.
```suggestion
            project_accessible = bool(task.project_id.id and self._document_check_access('project.project', task.project_id.id))
```
Sorry 😬